### ISSUE TITLE
Kaltura Media icon is not being displayed in TinyMCE editor

### ISSUE THREAD

**Issue by guillalva06** [kaltura/moodle_plugin#136]
In the last version of the plugin, the Kaltura icon for the TinyMCE editor was deleted. This is causing a javascript error when the editor tries to load the icon and obviously the icon is not been displayed on the editor. Is it possible to add the icon image again?, the image was in folder lib/editor/tinymce/plugins/kaltura/tinymce/img. I have attached an image showing how look the edito r without the icon.
<img width="982" alt="kaltura missing icon" src="https://cloud.githubusercontent.com/assets/11340590/25404171/889e8694-29c4-11e7-8532-9dea49c0114d.png">

**Comment by kaltura-hooks** [comment]
Hi @guillalva06,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa \"kaltura*\"
```	
	For deb based systems:
```	
	# dpkg -l \"kaltura-*\"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color \"ERR:\|PHP\|trace\|CRIT\|\[error\]\"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by kaltura-hooks** [comment]
Hi @guillalva06,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa \"kaltura*\"
```	
	For deb based systems:
```	
	# dpkg -l \"kaltura-*\"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color \"ERR:\|PHP\|trace\|CRIT\|\[error\]\"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by muli** [comment]
Hello @guillalva06,

Thank you for reporting this. Indeed, this is a bug and we will fix it in the upcoming release of our Moodle plugin.

In the meantime, if you have access to our plugin file on your server, you could fix this issue by creating a new directory, called 'img', under lib/editor/tinymce/plugins/kalturamedia/tinymce and copy the icon.png file from the 'pix' directory in the 'kalturamedia' directory.

I hope this helps, please let us know otherwise.

**Comment by muli** [comment]
Should be fixed in next plugin version (4.1).
